drivers/bridge/bridge.go

```diff
@@ -168,9 +168,23 @@ func (c *networkConfiguration) Conflict(o *networkConfiguration) bool {
 	return false
 }
 
+func (n *bridgeNetwork) isDefaultNetwork() bool {
+	if n.config.BridgeName == DefaultBridgeName {
+		return true
+	}
+	return false
+}
+
 // FromMap retrieve the configuration data from the map form.
-func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
+func (c *networkConfiguration) FromMap(d *driver, data map[string]interface{}) error {
```

Comment:
@aboch I see your point on passing driver to the FromMap utility method. Have changed that. But, with this requirement of using the driver-default configs, the parseXXX functions becomes driver specific and hence it is appropriate to make it a driver method. It feels wierd the other way around.